src/app/components/POCs/ProcessorTemplateSelector/ProcessorTemplateSelector.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Card,
+  CardBody,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Flex,
+  FlexItem,
+  Split,
+  SplitItem,
+  Text,
+  TextContent,
+  Title,
+} from "@patternfly/react-core";
+import { useTranslation } from "@rhoas/app-services-ui-components";
+import "./ProcessorTemplateSelector.css";
+import ProcessorCodeEditor from "@app/components/POCs/ProcessorCodeEditor/ProcessorCodeEditor";
+
+interface ProcessorTemplate {
+  icon?: React.ComponentType;
+  title: string;
+  description: string;
+  code: string;
+}
+
+export interface ProcessorTemplateSelectorProps {
+  onCancel: () => void;
+  onNext: (templateCode: string) => void;
+  onSkip: () => void;
+  templates: ProcessorTemplate[];
+}
+
+export const ProcessorTemplateSelector = ({
+  onCancel,
+  onNext,
+  onSkip,
+  templates,
+}: ProcessorTemplateSelectorProps): JSX.Element => {
+  const { t } = useTranslation("smartEventsTempDictionary");
+
+  const [selectedTemplate, setSelectedTemplate] = useState<ProcessorTemplate>(
+    templates[0]
+  );
+
+  return (
+    <Split className="processor-template-selector" hasGutter>
+      <SplitItem className="processor-template-selector__left-section">
```

Comment:
I'm seeing some weird behavior when resizing the window horizontally.
When reducing the size only the left column shrinks until it disappears, while the right column width stays the same.

I would expect both columns to resize, with the left one eventually keeping a minimum width at lower resolutions. 

I have the impression that it's tricky to manage columns widths settings with `Split`. 
Maybe a `Grid` could be a better fit? Changing the `span` config at low breakpoints? Wdyt?

Reply:
This should be solved with https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/291/commits/ff2e3924a9654a3e9f0a1ced95dbf90f53d955cf